### DISCUSSION TITLE
could AnySoftKeyboard became a swift keyboard?

### DISCUSSION THREAD

**Discussion by renato67** [AnySoftKeyboard/AnySoftKeyboard#2837] [Q&A]
I've read somewhere that there is a BETA project to let the keyboard became a swift keyboard. (that I will prefer…).
Do someone know the time for this project?

TIA
Renato

**Comment by adeyo50** [comment]
it exists in beta.  Go into the app settings and check the Enable gesture typing [BETA] checkbox
